src/js/event-target.js

```diff
@@ -119,6 +119,16 @@ EventTarget.prototype.one = function(type, fn) {
   this.addEventListener = ael;
 };
 
+EventTarget.prototype.any = function(type, fn) {
+  // Remove the addEventListener alialing Events.on
```

Comment:
"alialing"? typo?

Reply:
Yup, apparently it's everywhere else in the file as well, 😆